WPFUI/Common/SnapLayout.cs

```diff
@@ -81,6 +83,18 @@ private IntPtr HwndSourceHook(IntPtr hWnd, int uMsg, IntPtr wParam, IntPtr lPara
 
                     break;
 
+                case User32.WM.NCLBUTTONUP:
+                    if (_isButtonClicked)
```

Comment:
What about
```c#
if (!_isButtonClicked || !IsOverButton(wParam, lParam)) break;
```

Reply:
I think "if" need "{}".

Reply:
I edit it in GitHub, so I can’t make sure it has no error.